playbooks/group_vars/all.yml

```diff
@@ -15,7 +15,11 @@ xnat_home_dir: "{{ xnat_root_dir }}/home"
 database_server_certificate_cache_filename: "{{ ansible_cache_dir }}/pg_certificates/{{ xnat_db.host }}.pg.server.crt"
 database_client_certificate_cache_filename: "{{ ansible_cache_dir }}/pg_certificates/{{ xnat_db.host }}.pg.client.crt"
 
-postgresql_rpm_gpg_key_pgdg: "https://apt.postgresql.org/pub/repos/yum/keys/PGDG-RPM-GPG-KEY-RHEL"
+postgresql_rpm_gpg_key_pgdg:
+  x86_64: >-
+    https://apt.postgresql.org/pub/repos/yum/keys/PGDG-RPM-GPG-KEY-RHEL
+  aarch64: >-
+    https://apt.postgresql.org/pub/repos/yum/keys/PGDG-RPM-GPG-KEY-AARCH64-RHEL
```

Comment:
Okay, agreed about removing it.

Reply:
When you say at the inventory level, it would be ideal if this would work in either case. Like on my own laptop I want https://apt.postgresql.org/pub/repos/yum/keys/PGDG-RPM-GPG-KEY-AARCH64-RHEL, but production will want https://apt.postgresql.org/pub/repos/yum/keys/PGDG-RPM-GPG-KEY-RHEL.